Docs/sphinx_documentation/source/GPU.rst

```diff
@@ -350,8 +350,8 @@ Below is an example configuration for HIP on Tulip:
    cmake --build build -j 6
 
 
-Enabling SYCL support (experimental)
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Enabling SYCL support
```

Comment:
I think "support" should be capitalized as "Support". This way the titles remain consistent.

Reply:
```suggestion
Enabling SYCL Support
```